tests/test_pattern_rewriter.py

```diff
@@ -671,16 +687,20 @@ def test_inline_block_before():
   scf.if(%0 : !i1) {
     %1 : !i32 = arith.constant() ["value" = 2 : !i32]
   } {}
-} {}
+} {
+}
 }"""
```

Comment:
There is still an empty region here without a newline. Should this not also be expanded?

Reply:
I see, this is on the lhs of the rewrite so not what is generated by us, but what is parsed. 
No change is necessary here, but if we change this we should expand both regions (and the regions of all parsed programs here).